manual/src/tutorials/parallelism.etex

```diff
@@ -593,17 +593,17 @@ This program uses a shared reference cell protected by a mutex to communicate
 between the different systhreads running on two different domains. The
 systhread identifiers uniquely identify systhreads in the program. The initial
 domain gets the domain id 0 and thread id 0. The newly spawned domain gets
-the domain id as 1.
+the domain id 0.
```

Comment:
Is this correct? I believe this should remain `1`.

Reply:
Oh goodness, that's a typo! Thanks for catching it